### DISCUSSION TITLE
"Whole word deletion only" mode

### DISCUSSION THREAD

**Discussion by laralex** [monkeytypegame/monkeytype#2246] [Ideas] [answered]
Hi, I like to train my typing skill by always retyping misspelled words (i.e. in case of an error I do Ctrl+Backspace and type again). I would really like to have a mode (or toggle) on Monkeytype, that would enforce this behaviour (no plain Backspace, only Ctrl+Backspace).

Thanks in advance

**Comment by Miodec** [comment, marked as answer]
Or i mean... you could just build muscle memory to only use ctrl + backspace. Seems a bit niche to add it as a mode for everyone.

**Reply by laralex** [reply]
Yeah, I try, but from time to time I fall to plain old Backspace anyway. I just thought that such a mode isn't hard to implement, and it's not  as niche as say "stop on error" mode, which is inapplicable to the real typing (imho). I could even try to implement it myself, if you would point out where to look for

**Reply by Miodec** [reply]
I dont agree with stop on error not being used. Many people use it to recreate typeracer, where you have to get the word right to continue.
